sdk/storage/Azure.Storage.Blobs/BreakingChanges.txt

```diff
@@ -1,6 +1,10 @@
 Breaking Changes
 ================
 
+12.0.0-preview.2 (2019-08)
```

Comment:
Replace this line with `Next Release` instead of using the reference for the last release.